proteus/tests/mesh_tests/test_quads.py

```diff
@@ -79,7 +79,10 @@ def teardown_method(self,method):
     def test_01_FullRun(self):
         import filecmp
         self.ns.calculateSolution('test1')
-        relpath = "comparison_files/poiseulle_xmf.output"
+        if self.ns.ar[0].global_sync:
```

Comment:
@alistairbntl the reason I got differences here was that the new stack wasn't building h5py properly, which forces proteus.Archive to generate one xmf file per proc with slightly different metadata. I added back the old file and put in a check to see which file to use for comparison. I also fixed the stack so h5py is installed properly so the test should pass regardless of whether h5py is in the stack or not. BTW, we should probably be comparing the h5 files as well since that's where all the actual data is.  Might be able to use h5diff.